pkg/util/iswimi/iswimi.go

```diff
@@ -0,0 +1,14 @@
+package iswimi
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import "github.com/Azure/ARO-RP/pkg/api"
+
+// IsWimi checks whether a cluster is Workload Identity or classic
```

Comment:
```suggestion
// IsWimi checks whether a cluster is a workload identity cluster or a service principal cluster
```

I'm suggesting this change because "classic" refers to a non-HCP cluster as opposed to an HCP cluster. We will eventually have both _WIMI classic clusters_ and _WIMI HCP clusters_.